geowebcache/s3storage/src/main/java/org/geowebcache/s3/S3BlobStoreConfig.java

```diff
@@ -266,6 +278,30 @@ public String getProxyPassword() {
     public void setProxyPassword(String proxyPassword) {
         this.proxyPassword = proxyPassword;
     }
+    
+    /**
+     * Checks access type
+     *
+     * @return public or private access
+     */
+    public CannedAccessControlList getAccessControlList() {
+        CannedAccessControlList accessControlList;
+        if(access.equalsIgnoreCase("private")) {
+            accessControlList = CannedAccessControlList.BucketOwnerFullControl;
+        } else {
+            accessControlList = CannedAccessControlList.PublicRead;
+        }
+        return accessControlList;
+    }
+
+    /**
+     * Sets whether access should be private or public
+     *
+     * @param access whether access is private or public
+     */
+    public void setAccess(String access) {
+        this.access = access;
```

Comment:
A null check before setting this would be a good idea.  Is there a reason you went with a String for this rather than an Enum?